cea/interfaces/dashboard/lib/auth/providers.py

```
@@ -1,4 +1,5 @@
 import json
+import time
```

Ruff (0.8.2)

`time` imported but unused

Remove unused import: `time`

(F401)